### ISSUE TITLE
Stadia | stadia.google.com

### ISSUE THREAD

**Issue by Nalci** [PreMiD/Presences#4998]
### Description

I have a problem with the stadia presence. I installed it yesterday and when I'm on the stadia page it says on discord only "can't read page". When I'm in a game, it shows the game name correctly.

The problem exists also with PreWrap.

### Steps to Reproduce

1. Install the stadia presence
2. Go to stadia.google.com and Login
3. Go to home or browse the store and look at your status in Discord

### Expected Behavior

It should display the pages name where you on.

### Screenshots

![stadia_presence_1](https://user-images.githubusercontent.com/95542245/144763528-c0dcc77e-b110-4dcc-98b4-a0f7a3d463ba.JPG)

### Operating System (OS)

Windows 10

### Browser

Chrome

### Browser Version

Version 96.0.4664.45 (Offizieller Build) (64-Bit)

**Comment by Pjoottr** [comment]
Sorry, I can't seem to reproduce this. Maybe try [refreshing your cache](https://support.google.com/accounts/answer/32050)?

Version 96.0.4664.93 (Official Build) (64-bit)

![image](https://user-images.githubusercontent.com/93800318/144932505-8c856326-cfc7-4b46-9688-91de41ee0c50.png)

**Comment by Nalci** [comment]
Hello,

thank you for the reply!

I deleted all cache and data but nothing changed. I messed a bit aroung because it really wondered me why it worked for you but not for me.

I noticed some things:
1. If I have only one Google Account active, the URL is "https://stadia.google.com/home" and it works correctly
2. If I have multiple Google Accounts active and choose the one for Stadia, it changes to "https://stadia.google.com/u/1/home" - it shows "can't read page". But if I open "https://stadia.google.com/u/1/store" it shows "Viewing Stadia Store" correctly. Maybe it's a URL thing?
3. If I browse my library or click on discovery it shows again "can't read page". Is it maybe possible, to add also "Viewing Library" or "Browsing DIscovery" to it?

I hope it's not too confusing, english is not my native language.

Thank you very much!
